packages/ui/src/types/types.ts

```diff
@@ -0,0 +1,3 @@
+import { BreakPoint } from '../themes/screens.styled';
+
+export type MediaQuery<T> = Partial<Record<BreakPoint, T>> | T;
```

Comment:
We don't need a folder for types

Reply:
We will be needing that as `index.d.ts` file should be in another folder as it is conflicting with the `index.ts` file in the `src` folder.

Reply:
- Move the `types/types.ts` to `types.ts` 
- Or Move the `types/types.ts` to `types/index.ts`

Reply:
Done